test/integration/ethereum/flashMintLeveragedExtended.spec.ts

```diff
@@ -0,0 +1,706 @@
+import "module-alias/register";
+import { Account, Address } from "@utils/types";
+import DeployHelper from "@utils/deploys";
+import { getAccounts, getWaffleExpect, preciseMul } from "@utils/index";
+import { setBlockNumber } from "@utils/test/testingUtils";
+import { ethers } from "hardhat";
+import { BigNumber, utils } from "ethers";
+import { FlashMintLeveragedExtended } from "../../../typechain";
+import {
+  IWETH,
+  StandardTokenMock,
+  IDebtIssuanceModule,
+  IERC20__factory,
+  AaveV3LeverageStrategyExtension__factory,
+} from "../../../typechain";
+import { PRODUCTION_ADDRESSES, STAGING_ADDRESSES } from "./addresses";
+import { ADDRESS_ZERO, MAX_UINT_256 } from "@utils/constants";
+import { ether } from "@utils/index";
+import { impersonateAccount } from "./utils";
+
+const expect = getWaffleExpect();
+
+enum Exchange {
+  None,
+  Sushiswap,
+  Quickswap,
+  UniV3,
+  Curve,
+}
+
+type SwapData = {
+  path: Address[];
+  fees: number[];
+  pool: Address;
+  exchange: Exchange;
+};
+
+if (process.env.INTEGRATIONTEST) {
+  describe.only("FlashMintLeveragedExtended - Integration Test", async () => {
+    const addresses = process.env.USE_STAGING_ADDRESSES ? STAGING_ADDRESSES : PRODUCTION_ADDRESSES;
+    let owner: Account;
+    let deployer: DeployHelper;
+
+    let rEth: StandardTokenMock;
```

Comment:
this works for now but lets add some basic arbitrum integration tests in later PR's